src/pages/home/home.scss

```
@@ -0,0 +1,27 @@
+.form {
+  display: flex;
+  flex-flow: column wrap;
+  width: 340px;
+  min-height: 450px;
+  padding: 50px 30px 30px;
+  background: #FFFFFF;
+  box-shadow: 0 0 6px rgba(0, 0, 0, 0.14);
+  border-radius: 12px;
+  text-align: center;
+
+  &__inputsbox {
```

Словосочетания лучше разбивать дефисом: 
```suggestion
  &__inputs-box {
```